Update docs for the upcoming mixed legacy/spack release (#337)
FairRootGroup/FairSoft: legacy/README.md

# FairSoft (Legacy)

## Preface

> This will be the last release featuring the bash/cmake
> based setup system. Upcoming releases are planned to use
> spack instead (for more information see
> [here](../docs/README.md)).
> 
> The remainder of this README is basically the main README
> from the last releases. All file references should be read
> as relative to the `/legacy/` folder.

Reviewer: ```suggestion
Our homegrown bash/cmake based setup system
has been named "Legacy". It has been moved to the
sub-directory `legacy/` to distinguish it clearly
from the future Spack-based setup system
(for more information see [here](../docs/README.md)).
The latter will eventually replace the "Legacy" setup system
in a future release.
```

* I prefer to advertize the upcoming spack-only release "less final". Sure, right now it looks like it is going to be the next release in half a year. But what if we have to release another time in between, which we do not know yet today.
* Let's delete the note on references and rather update such references everywhere please.

## Guided Installation

To start the installation procedure one has to run the configure.sh script which is

> found in the current directory of FairSoft from within this directory. If the script is

Reviewer: ```suggestion
found in the `legacy/` directory of FairSoft. If the script is
```

called without parameters one is guided through some menus to choose the appropriate
setup.

```

> $ ./configure.sh

Reviewer: ```suggestion
legacy/$ ./configure.sh
```
